src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { FilterParam } from '../types/FilterParam';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[];
+  isOneTodoCompleted: boolean;
+  filterParam: FilterParam;
+  setFilterParam: (newValue: FilterParam) => void;
+  deleteCompletedTodos: () => void;
+};
+
+export const TodoFooter: React.FC<Props> = ({
+  todos,
+  isOneTodoCompleted,
+  filterParam,
+  setFilterParam,
+  deleteCompletedTodos,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${todos
+          .filter(({ completed }) => completed === false).length} items left`}
```

Comment:
If there is one item, there should not be a plural in the text.
It is also worth transferring the calculation to a separate variable.

Reply:
Not fixed